napari_allencell_annotator/assets/install_napari.command

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
```

Comment:
I thought we were using zsh for this?

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/97/commits/83bbf28390b071bb9d89bd14db8357f40ae05a7d